Formula/g/github-release.rb

```diff
@@ -25,6 +25,7 @@ def install
     assert_match "github-release v#{version}", shell_output("#{bin}/github-release --version")
 
     system bin/"github-release", "info", "--user", "github-release",
-                                            "--repo", "github-release"
+                                            "--repo", "github-release",
+                                            "--tag", "v#{version}"
```

Comment:
```suggestion
                                         "--repo", "github-release",
                                         "--tag", "v#{version}"
```

Reply:
Agree, alignment could be better. I just put back what I removed in
* https://github.com/Homebrew/homebrew-core/pull/230899